docs(reference): add security notice 14 (#5150)
camunda/camunda-docs: versioned_docs/version-8.6/reference/notices.md

Camunda publishes security notices after fixes are available.

### Notice 15

#### Publication date

March 11th, 2025

#### Product affected

Camunda Optimize

#### Impact

> The version of Camunda Optimize was affected by a vulnerability that allows a malicious attacker to craft Camunda URLs that could execute JavaScript code.

Reviewer: ```suggestion
Some Camunda Optimize versions were affected by a vulnerability that allowed a malicious attacker to craft Camunda URLs that could execute JavaScript code.
```
See previous comment
